app/controllers/admins/employees_controller.rb

```diff
@@ -27,8 +27,32 @@ def destroy
       redirect_back fallback_location: kudos_path
     end
 
+    def add_kudos_form
+      render 'add_kudos_form'
+    end
+
+    def add_kudos
+      kudos_number = permit_add_kudos[:number_of_available_kudos_to_add].to_i
+      unless kudos_number.between?(1, 20)
+        flash[:alert] = 'Number of additional kudos must be in range 1 to 20.'
+        render 'add_kudos_form'
+        return
+      end
+      Employee.add_kudos_to_all(kudos_number)
```

Comment:
In my opinion, it is better to have all the logic of adding kudos in one place, therefore I would put adding kudos in here, not in the Employee model. Especially when there is no need to call `Employee.add_kudos_to_all` from any other place.

Reply:
Thanks for comments. 
I had hard time deciding where to put it, at the end I have no preference. It did seem like a model responsibility so I put it there. However I do agree that putting it here makes it easier to read.